bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/virtual/ButtonVirtualDatapointHandler.java

```diff
@@ -36,6 +36,8 @@
     private static final String LONG_REPEATED_EVENT = "LONG_REPEATED";
     private static final String LONG_RELEASED_EVENT = "LONG_RELEASED";
 
+    private boolean usesLongStartEvent;
```

Comment:
Probably need a better solution here, as the handler is a singleton. Either a `Map<HmDevice, Boolean>` or some other way to suppress the start button press timeout handling (LONG on HM, LONG_START on HMIP) without suppressing the repetition timeout handling (which is LONG on HMIP).